src/main/java/org/opentripplanner/graph_builder/issues/InvalidVehicleParkingCapacity.java

```diff
@@ -0,0 +1,26 @@
+package org.opentripplanner.graph_builder.issues;
+
+import org.opentripplanner.graph_builder.DataImportIssue;
+
+public class InvalidVehicleParkingCapacity implements DataImportIssue {
+
+  public static final String FMT = "Capacity for osm node %d is not a number: '%s'; it replaced with '-1' (unknown).";
```

Comment:
There is a tiny typo in there: should be "it's" or "it will be".

Reply:
Thanks, fixed.